.github/workflows/generate-preview.yml

```diff
@@ -21,6 +21,7 @@ jobs:
         uses: ruby/setup-ruby@v1
         with:
           ruby-version: 2.6
+          bundler: 1.17.2
```

Comment:
Maybe add a comment above this saying that unpinning this breaks i18n image links? Until we know why, we should make sure nobody "cleans" this up.